core/designsystem/src/main/java/com/mashup/dorabangs/core/designsystem/component/card/FeedCard.kt

```diff
@@ -41,7 +45,28 @@ fun FeedCard(
     onClickCardItem: () -> Unit = {},
     onClickBookMarkButton: () -> Unit = {},
     onClickMoreButton: () -> Unit = {},
+    updateCardState: () -> Unit = {},
 ) {
+    val loadingSecond = if (!cardInfo.createdAt.isNullOrBlank()) {
+        cardInfo.createdAt.convertCreatedSecond()
+    } else {
+        8
+    }.between(0, 8)
+
+    LaunchedEffect(cardInfo.isLoading) {
+        var requested = false
+        while (cardInfo.isLoading) {
+            val currentLoadingSecond = if (requested) {
+                4000
+            } else {
+                requested = true
+                loadingSecond * 1000L
+            }
+            delay(currentLoadingSecond)
+            updateCardState()
+        }
+    }
```

Comment:
```suggestion
    LaunchedEffect(cardInfo.isLoading) {
        if (cardInfo.isLoading) {
            delay(8000L)
            updateCardState()
        }
    }
```
위에 애니메이션 8초짜리 적용한다는 가정하에, 우리 폴링은 8초뒤에 날리기로 했던걸로 기억나는뎅 
그럼 굳이 다른거 다없애고 8초뒤에 update하문 되는거 아잉가 ?? 4초에 한번 체크하는 로직인건가유 ?

맞다면 
delay(4000L)
체크
delay(4000L) 은 어떠신지,,

Reply:
@Ahn-seokjoo 
최초 8초 이후 한 번 확인하고, 다음거 확인할 때 8초뒤에 하면 너무 긴 것 같아서 임의로 4초로 넣엇음...

Reply:
나도 여기 4초 로직 이해가 안가는데 설명해줄 수 있나용